src/utils/helper-functions.tsx

```diff
@@ -95,8 +101,12 @@ export const getFormattedDate = ({
   isCurrentDayHours = false,
   includeOrdinal = false
 }: FormatedDate): string => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const { i18n } = useTranslation()
```

Comment:
We can't ignore this rule, you need to fix that

![image](https://github.com/user-attachments/assets/1b5ac9d2-5d33-48f8-85be-68a37aba9d6f)

[React documentation](https://react.dev/warnings/invalid-hook-call-warning#breaking-rules-of-hooks)